.eslintrc.json

```diff
@@ -1,3 +1,71 @@
 {
```

Comment:
Most places we have removed the `.eslintrc.json` file and replaced it with a `.eslintrc.js` file like [this one](https://github.com/salesforcecli/plugin-template-sf/blob/main/.eslintrc.js). You can still add custom rules to that if needed ([example](https://github.com/salesforcecli/plugin-release-management/blob/580aabf621deed609ebc9afbd5cbc0ef191a40ba/.eslintrc.js#L9))

Reply:
Will do! I'd like to have you re-review the state of the .js file once I push the update, to make sure it all looks valid.